plugins/jquery.js

```diff
@@ -72,4 +72,19 @@ $.ajax = function ravenAjaxWrapper(url, options) {
     }
 };
 
+var _oldDeferred = $.Deferred;
+$.Deferred = function ravenDeferredWrapper(func) {
```

Comment:
Worst case is that it introduces (a broken) `$.Deferred` when it wasn't there before. But yeah, I'd probably leave it if it's not defined.

Aside, `$.Deferred` was introduced in [jQuery 1.5](https://api.jquery.com/category/version/1.5/) in 2011.